01-backup.md

```diff
@@ -54,6 +54,23 @@ we're telling Git:
 
 The four commands above only need to be run once:
 the flag `--global` tells Git to use the settings for every project on this machine.
+The configuration options get stored in your home directory in a text file called `.gitconfig`.
+
+~~~ {.bash}
+$ cat ~/.gitconfig
+~~~
+~~~ {.output}
+[user]
+	name = Vlad Dracula
+	email = vlad@tran.sylvan.ia
+[color]
+	ui = true
+[core]
+	editor = nano
+~~~
+
+This file can be edited directly.
```

Comment:
We've omitted mention of `.gitconfig` from the novice lesson 
deliberately - too much detail too early.  Can this and other notes 
(including anything to do with branching - see the second para of 
`instructors.md`) please be put in a new file `05-extra.md` with 
information for learners who've gone further and faster than our basic 
lesson?